recipes/libcurl/all/conanfile.py

```diff
@@ -460,12 +453,12 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
-        self.cpp_info.names["cmake_find_package"] = "CURL"
-        self.cpp_info.names["cmake_find_package_multi"] = "CURL"
-        self.cpp_info.names["pkg_config"] = "libcurl"
-        self.cpp_info.components["curl"].names["cmake_find_package"] = "libcurl"
-        self.cpp_info.components["curl"].names["cmake_find_package_multi"] = "libcurl"
-        self.cpp_info.components["curl"].names["pkg_config"] = "libcurl"
+        self.cpp_info.set_property("cmake_find_package", "CURL")
+        self.cpp_info.set_property("cmake_find_package_multi", "CURL")
+        self.cpp_info.set_property("pkg_config", "libcurl")
+        self.cpp_info.components["curl"].set_property("cmake_find_package", "libcurl")
+        self.cpp_info.components["curl"].set_property("cmake_find_package_multi", "libcurl")
+        self.cpp_info.components["curl"].set_property("pkg_config", "libcurl")
```

Comment:
Careful with the property 'key'. The new names are a bit different.
```suggestion
        self.cpp_info.set_property("cmake_target_name", "CURL")
        self.cpp_info.set_property("pkg_config_name", "libcurl")
        self.cpp_info.components["curl"].set_property("cmake_target_name", "libcurl")
        self.cpp_info.components["curl"].set_property("pkg_config_name", "libcurl")
```

Reply:
We want to keep `cmake_find_package` and `cmake_find_package_multi`.
conan-center-index recipes needs to cater both conan users and non-conan users.

By non-conan users, I mean projects we package in cci that don't use conan.

Reply:
Maybe I'm wrong, but 
https://docs.conan.io/en/latest/reference/conanfile/attributes.html?highlight=set_property
seems to imply that new properties are
```
cmake_file_name property will affect all cmake generators that accept the filenames property (cmake_find_package and cmake_find_package_multi).
cmake_target_name property will affect all cmake generators that accept the names property (cmake, cmake_multi, cmake_find_package, cmake_find_package_multi and cmake_paths).
cmake_build_modules property will replace the build_modules property.
pkg_config_name property will set the names property for pkg_config generator.
```
and thus providing `cmake_find_package` could yield a NO-OP.

Reply:
You're right. I was under the impression `cmake_find_package` was being removed.